ui/apps/bookmarks/src/components/bookmarks-page.tsx

```diff
@@ -119,7 +119,7 @@ const BookmarksPage: React.FC<BookmarksPageProps> = props => {
               )}
             />
           )}
-          {bookmarksReq.status === 'success' && bookmarks && (
+          {bookmarksReq.status === 'success' && bookmarks.length > 0 && (
```

Comment:
this can be simplified as `bookmarksReq.status === 'success' && bookmarks?.length && ( `

Reply:
I did that on purpose otherwise this will be how it gets rendered when bookmarks is empty that is using `bookmarksReq.status === 'success' && bookmarks?.length`

<img width="699" alt="image" src="https://user-images.githubusercontent.com/35398733/191910488-606de852-b7b4-4206-9a7d-59c4666981af.png">